### PR TITLE
Removing Libvirt

### PR DESCRIPTION
**Pull request by larrydewey**
A significant amount of legacy code existed around an attempt to attest
the state of the host machine. As this causes a bug while using
`sys_info()`, this opts to remove all of this legacy code to help keep
the tool cleaner and limit possible bugs.

Signed-off-by: Larry Dewey <larry@SCS-L10-lardewe.amd.com>

### CODE REVIEW

File: deps-install.sh [hunk 1]
```diff
@@ -21,7 +21,6 @@ NEED_DEPS=0
 INSTALLER=""
 SSL_DEV=""
 AUTO_CONF="autoconf"
-LIBVIRT_NAME="libvirt-devel"
```
Comment (sphrasavath) [inline deps-install.sh:24]: looks good here

**Review by sphrasavath — COMMENTED** [review]
Just a question, does sev-tool rely on other components like qemu or ovmf versions to get info from psp fw?  My guess is no, but how do we currently validate that sev is supported on the system prior to use?

**Comment by larrydewey** [comment]
@sphrasavath originally this was added to make sure the machine was in a good state, but it relied heavily on using libvirt and a custom built instance of OVMF to verify things were working properly. As the means of checking this no longer is valid (in some cases), it made sense to just remove this.

**Comment by larrydewey** [comment]
> @larrydewey I'm assuming that we could do something like SEV init or SNP init and should take care of any state checking w/out libvirt? If yes, I'm good with this change.

I don't see why not. I will go ahead and merge these changes in.

**Pull request merged**